### DISCUSSION TITLE
Would like to filter column names by data type

### DISCUSSION THREAD

**Discussion by GTGeek88** [dbeaver/dbeaver#21409] [Ideas]
This is in regards to this column filtering option: https://dbeaver.com/docs/dbeaver/Configure-Filters/

It'd be nice to be able to filter by data type, so that I could see, for instance, only the datetime fields in a table.

**Comment by LonwoLonwo** [comment]
Hello

At least, you can click on the data type column to sort columns by data types names.

Maybe it will be helpful.

![2023-10-06 18_06_12-DBeaver Enterprise 23 2 2 - Invoice](https://github.com/dbeaver/dbeaver/assets/45152336/89f67522-343f-4bf1-b17e-81aaaf27510e)
